src/app/core/_models/file.model.ts

```diff
@@ -1,26 +1,52 @@
-import { AccessGroup } from './access-group.model'
+import { AccessGroup } from './access-group.model';
 
+export enum FileType {
```

Comment:
Im using something similar in core/_constants/files.config

Reply:
Yes I know, this enum will replace that one was my thought.

Reply:
Yes, that sounds perfect.